lib/src/client.dart

```diff
@@ -6,17 +6,19 @@ import "package:xconn/src/session.dart";
 import "package:xconn/src/types.dart";
 
 class Client {
-  Client({IClientAuthenticator? authenticator, Serializer? serializer}) {
+  Client({IClientAuthenticator? authenticator, Serializer? serializer, Duration? keepAliveInterval}) {
```

Comment:
Instead of directly passing the keepAlive interval, that should be part of a class "Config" maybe?